src/entities/modal-window/types.ts

```diff
@@ -0,0 +1,8 @@
+import type { Ref } from 'vue'
+
+export interface Modal {
```

Comment:
Типы, которые мы создаём сами, лучше называть не менее чем в два слова. Чтобы не путать их с уже существующими.

```suggestion
export interface ModalState {
```